baitap-submit/ten_cua_ban/01-prompt-engineering/01-prompt-engineering.md

```diff
@@ -1,11 +1,49 @@
-- Prompt 1
+- Prompt 1 . Đưa nội dung bài học vào	Tạo danh sách các câu hỏi trắc nghiệm:
```

Comment:
Nội dung bài học e nên để như là template dạng như {{TEXT}} và dc bao bởi ký tự như === hoặc """

Thứ 2 nên nêu rõ chi tiết e muốn câu hỏi như thế nào? ví dụ cậu hỏi khó, dễ. 
Ví dụ như vầy 

Please generate a list of multiple-choice questions based on the provided lesson content. The questions should assess comprehension and critical thinking related to the material.

Guidelines for the questions:

- Each question should have four answer options (A, B, C, D), with one correct answer and three distractors.
- Ensure that the questions cover various aspects of the lesson, including key concepts, important details, and any significant themes.
- Aim for clarity and conciseness in both the questions and the answer choices.
- Indicate the correct answer for each question at the end of the list.

===
{{TEXT}}
===